extensions/src/AWSSDK.Extensions.CloudFront.Signers/AWSSDK.Extensions.CloudFront.Signers.NetFramework.csproj

```diff
@@ -1,10 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net472</TargetFramework>
 	<AssemblyName>AWSSDK.Extensions.CloudFront.Signers</AssemblyName>
 	<PackageId>AWSSDK.Extensions.CloudFront.Signers</PackageId>
-	<Version>4.0.1.0-preview</Version>
```

Comment:
`Version` and `Title` are set in the `.nuspec` file, so these values here are ignored.